app/services/curator/digital_object_factory_service.rb

```diff
@@ -109,7 +109,7 @@ def find_or_build_collection_members!(digital_object, admin_set_ark_id, collecti
 
           collection = Curator.collection_class.select(:id, :ark_id).find_by!(ark_id: collection_ark_id)
 
-          next if digital_object.collection_members.exists?(collection: collection)
+          next if digital_object.new_record? || digital_object.collection_members.exists?(collection: collection)
```

Comment:
I'm confused by this. Why would we skip `digital_object.collection_members.build` if this is a new record?